### DISCUSSION TITLE
The site configured at this address does not contain the requested file

### DISCUSSION THREAD

**Discussion by GoldenOblum** [community/community#23856] [New to GitHub]
<p>I’m new to GitHub, recently I created my first ever repository and uploaded files from my computer into the repository I created, I just basically dragged and dropped the entire project folder and followed along all the steps in the tutorial. Now that I’m trying to visit the webpage, I’m receiving the following message ’ 404, file not found, the site configured at this address does not contain the requested file. If this is your site, make sure that the filename case matches the URL.<br>
For root URLs (like <code>http://example.com/</code> ) you must provide an <code>index.html</code> file.’</p>
<p>Can anyone tell me what the issue is?</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/5/c5071e62a9c373f2288ba66b433a1af5b558cd9b.png" data-download-href="/uploads/short-url/s6ZjqJRrvOOBpt4jN1bJ0CM4Vl1.png?dl=1" title="GitHub" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/90712505/181079727-6766ac73-954c-4b4a-a195-749817e61720.png" alt="GitHub" data-base62-sha1="s6ZjqJRrvOOBpt4jN1bJ0CM4Vl1" width="690" height="256" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c5071e62a9c373f2288ba66b433a1af5b558cd9b_2_690x256.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c5071e62a9c373f2288ba66b433a1af5b558cd9b_2_1035x384.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/5/c5071e62a9c373f2288ba66b433a1af5b558cd9b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c5071e62a9c373f2288ba66b433a1af5b558cd9b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">GitHub</span><span class="informations">1323×492 34.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment]
<p>From the screenshot it looks like you put your files into a subdirectory. Whatever you want to show up when someone visits your <code>github.io</code> domain must be at the top level of your repository.</p>

**Comment by GoldenOblum** [comment]
<p>By subdirectory you mean Project I folder?</p>

**Comment by airtower-luna** [comment]
<p>Yes, exactly. What’s in there should be visible if someone adds “Project I” to the path, though.</p>

**Comment by GoldenOblum** [comment, marked as answer]
<p>Okay I’ll try doing it that way. Thanks</p>
